project/src/main/webapp/index.js

```diff
@@ -1,14 +1,31 @@
 window.onload = function onLoad() {
-  addLoginUrlToButton();
+  redirectIfLoggedIn();
 };
 
-/**
- * Adds login url to login button.
- */
-async function addLoginUrlToButton() {
+async function redirectIfLoggedIn() {
+  const loggedIn = await isLoggedIn();
+  if (loggedIn) {
+    window.location.href = 'events-feed.html';
+  } else {
+    addLoginUrlToElement('login-prompt');
+    addLoginUrlToElement('explore-link');
+  }
+}
+
+async function addLoginUrlToElement(id) {
+  const loginUrl = await getLoginUrl();
+  const element = document.getElementById(id);
+  element.href = loginUrl;
+}
+
+async function isLoggedIn() {
+  const response = await fetch('/login-status');
+  const loginStatus = await response.json();
+  return loginStatus.isLoggedIn;
```

Comment:
change to (loginStatus === "LOGGED_IN") due to change in PR #65

Reply:
Done